Website/src/rendering/src/components/Forms/AddressBookForm.tsx

```diff
@@ -151,7 +106,7 @@ const AddressBookForm = (props: AddressBookFormProps): JSX.Element => {
   const buttonText = props.isEditing ? 'Save Address' : 'Create address';
 
   return (
-    <form onSubmit={handleFormSubmit} className="form">
+    <form onSubmit={handleFormSubmit} className="form" ref={formRef}>
```

Comment:
Seems great overall, only issue I've encountered is when trying to create a new address directly from the address book, I get this error 

![image](https://user-images.githubusercontent.com/96472119/181229894-db559bbb-1e51-495c-9f64-b3e159151fe6.png)

no matter if the checkboxes below are checked or not.